.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         include:
           - os: macos-10.15
             wasm: false
-            CHECK_COMMAND: cargo check --all-targets
+            CHECK_COMMAND: cargo check --all-targets --features vulkan-portability
```

Comment:
Should we only be enabling this feature on mac/osx?

Reply:
Yes, this is the "- os: macos-10.15" section :)

Reply:
Oh the diff didn't show me that :)